apps/central/src/App.vue

```diff
@@ -28,12 +28,11 @@ export default {
           label: "Styleguide (for developers)",
           href: "/apps/styleguide/",
         },
-        // remove docs until available
-        // {
-        //   label: "Docs",
-        //   href: "/apps/docs/",
-        //   newWindow: true,
-        // },
+        {
+          label: "Docs",
+          href: "/apps/docs/",
+          newWindow: true,
```

Comment:
missing visual indication that a new tab is opened (see: https://www.w3.org/TR/WCAG20-TECHS/G201.html)

Reply:
shall we then for now set newWindow: false?

Reply:
that would be my personal pref , but 'new window' icon for all links opening a new window could also work.